src/NHSD.BuyingCatalogue.Ordering.Api/appsettings.json

```diff
@@ -1,4 +1,7 @@
 {
+  "IssuerUrl": "http://host.docker.internal:5104",

```

Comment:
This should be set to the Identity.Api. According to the Organisations.Api this is set as `http://localhost:5102/identity`.

Reply:
👍